pkg/operator/cluster/spec.go

```diff
@@ -39,3 +39,31 @@ type Spec struct {
 	// The path on the host where config and data can be persisted.
 	DataDirHostPath string `json:"dataDirHostPath"`
 }
+
+type PoolSpec struct {
```

Comment:
is the convention to put all Specs into a single file? i have my storageSpec in a storage related go file right now.